### ISSUE TITLE
1.16.5 server just randomly crashed 

### ISSUE THREAD

**Issue by s1h4d0w** [lucko/LuckPerms#2914]
### Description
Paper 1.16.5 server with LuckPerms 5.2.76 randomly crashed

### Reproduction steps
No idea, it randomly happened and has never happened before since running this server for a couple of years

### Expected behaviour
It not crashing

### Environment details
Paper 1.16.5 build 472
LuckPerms version 5.2.76

Log paste: https://www.lepelcraft.nl/log.txt
I used our own website as it was too big for pastebin

**Comment by Fefo6644** [comment]
This doesn't seem like a LuckPerms issue but more like a problem related to entities in your server, I suggest asking in the [PaperMC discord](https://discord.gg/papermc) or [making an issue in their repo](https://github.com/PaperMC/Paper/issues/new/choose).

But just to make sure, does it stop happening when you remove LuckPerms from the server?

**Comment by s1h4d0w** [comment]
I've only honestly seen this happening once, there was 1 player online, I was talking to them and just bam suddenly hundreds of errors in console until I managed to restart the server. I asked the player what they were doing but they said they were just standing still in the middle of nowhere talking to me.

I just saw it happen, was confused and decided to post here as it started off saying a luckperms process was hanging.
